style.css

```diff
@@ -97,6 +150,307 @@ h2 {
   line-height: 44px;
 }
 
+.works .work-items .work-item .work-img img {
+  max-width: 100%;
+  display: block;
+  margin: auto;
+}
+
+.works {
+  margin-top: 114px;
+  padding: 0 24px;
+}
+
+.works .work-items {
+  list-style: none;
+  display: -ms-grid;
+  display: grid;
+  row-gap: 88px;
+}
+
+.works .work-items .work-item {
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 16px 16px 28px 16px;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  max-width: 580px;
+  margin: auto;
+}
+
+.works .work-items .work-item .work-info h2 {
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 12px;
+  margin-bottom: 20px;
+}
+
+.works .work-items .work-item .work-info .work-info-details {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  gap: 12px;
+}
+
+.works .work-items .work-item .work-info .work-info-details .company-name {
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.works .work-items .work-item .work-info .work-info-details .position {
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.works .work-items .work-item .work-info .work-info-details .date {
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.works .work-items .work-item .work-info .project-description {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 20px;
+  margin-bottom: 12px;
+}
+
+.works .work-items .work-item .work-info .project-technologies {
+  display: -webkit-box;
+  display: flex;
+  gap: 8px;
+  margin-bottom: 24px;
+  list-style: none;
+}
+
+.works .work-items .work-item .work-info .project-technologies li {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  padding: 4px 12px;
+  color: #6070ff;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.works .work-items .work-item .work-info .see-project {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  display: inline-block;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+}
+
+.works .work-items .work-item .work-info .see-project:hover {
+  background-color: #396df2;
+  color: #fff;
+  -webkit-transition: 0.3s;
+  transition: 0.3s;
+}
+
+@media screen and (min-width: 767.98px) {
```

Comment:
- I am assuming this media query you are giving for ipad version. If that so, it's okay. (just make that `768` instead of `767.98px`)

- [ ] The problem is that you are using the same (`@media screen and (min-width: 767.98px) {.....}`) media query twice on line 869. Please put all the style for same resolution in one media query rather than making multiple. Follow DRY (Do not repeat yourself) principle.